docs/vcpkg-schema-definitions.schema.json

```diff
@@ -0,0 +1,715 @@
+        "repository": {
+          "type": "string",
+          "description": "The URI or SSH entry to clone from. (as if by `git clone`)"
+        },
+        "baseline": {
+          "description": "The SHA that will be checked out to find default versions for ports from this registry.",
+          "$ref": "#/definitions/sha1"
+        },
+        "ref": {
+          "type": "string",
+          "description": "The ref to fetch when cloning this git registry."
+        }
+      },
+      "required": [
+        "kind",
+        "repository",
+        "baseline"
+      ]
+    },
+    "has-schemed-version": {
+      "type": "object",
+      "oneOf": [
+        {
+          "properties": {
+            "version-string": {
+              "description": "Text used to identify an arbitrary version",
+              "type": "string",
+              "pattern": "^[^#]+$"
+            }
+          },
+          "required": [
+            "version-string"
+          ]
+        },
+        {
+          "properties": {
+            "version": {
+              "description": "A relaxed version string (1.2.3.4...)",
+              "type": "string",
+              "pattern": "^\\d+(\\.\\d+)*(?:-((?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*)(?:\\.(?:0|[1-9]\\d*|\\d*[a-zA-Z-][0-9a-zA-Z-]*))*))?(?:\\+([0-9a-zA-Z-]+(?:\\.[0-9a-zA-Z-]+)*))?$"
+            }
+          },
+          "required": [
+            "version"
+          ]
+        },
+        {
+          "properties": {
+            "version-semver": {
+              "$ref": "#/definitions/semantic-version"
+            }
+          },
+          "required": [
+            "version-semver"
+          ]
+        },
+        {
+          "properties": {
+            "version-date": {
+              "description": "A date version string (e.g. 2020-01-20)",
+              "type": "string",
+              "pattern": "^\\d{4}-\\d{2}-\\d{2}(\\.\\d+)*$"
+            }
+          },
+          "required": [
+            "version-date"
+          ]
+        }
+      ]
+    },
+    "identifier": {
+      "type": "string",
+      "description": "Identifiers used for port and feature names.",
+      "allOf": [
+        {
+          "description": "Identifier are lowercase with digits and dashes.",
+          "pattern": "[a-z0-9]+(-[a-z0-9]+)*"
+        },
+        {
+          "not": {
+            "description": "Identifiers must not be a Windows filesystem or vcpkg reserved name.",
+            "pattern": "^(prn|aux|nul|con|lpt[1-9]|com[1-9]|core|default)$"
+          }
+        }
+      ]
+    },
+    "maintainers": {
+      "description": "An array of strings which contain the authors of a package",
+      "$ref": "#/definitions/string-or-strings"
+    },
+    "override": {
+      "description": "A version override.",
+      "type": "object",
+      "properties": {
+        "name": {
+          "$ref": "#/definitions/identifier"
+        },
+        "version": {
+          "$ref": "#/definitions/version-text"
```

Comment:
I believe we now accept port-version as part of this field, which isn't permitted by `version-text`.